src/__tests__/Reducer.test.js

```diff
@@ -0,0 +1,10 @@
+// import country from '../__mock__/Country';
+import store from '../redux/store';
+// import covidSlice, { fetchCountries, fetchTotalData } from '../redux/covidSlice';
+
+describe('country redux state test', () => {
+  it('Should initially set county to an empty array', () => {
+    const state = store.getState().covid;
+    expect(state.countries).toEqual([]);
+  });
+});
```

Comment:
- [ ] Great work attempting to write unit tests for the reducers as requested by the previous reviewer. However, you are to write tests for the reducer and all other pure functions you have in the `redux/covidSlice.js` file. Kindly implement it properly 👍 .

Reply:
Added tests